src/types/Match.ts

```diff
@@ -12,5 +13,5 @@ export interface Match {
   startedAt: Dayjs | string
   endedAt: Dayjs | string | null
   notes: string
-  participants: Participant[]
+  participants?: Participant[]
```

Comment:
The participants are actually required in reality, if you look at the API. I suspect that you changed the wrong place here, you might need to think a bit on why you did this